.gitignore

```diff
@@ -12,7 +12,7 @@ tenderly.yaml
 .DS_Store
 
 #Hardhat files
-packages/dma-contracts/abi/generated
+packages/dma-contracts/abis/system/*
```

Comment:
@BackToTheCode Is this path correct? For me it looks more like: `packages/abis/system/*`
Following that I would also remove all already commited files in that path as it doesn't make sense to keep them as they are not up to date anyway. What you think?